src/cli/move.ts

```diff
@@ -310,6 +310,22 @@ export class Move {
     return this.runCommand(cliArgs, showStdout);
   }
 
+  async gasProfile(args: {
+    network: string;
+    transactionId: string;
+    extraArguments?: Array<string>;
+    showStdout?: boolean;
+  }): Promise<{ output: string; result?: any }> {
+    const { network, transactionId, extraArguments, showStdout } = args;
+    const cliArgs = ["aptos", "move", "replay", "--profile-gas", "--network", network, "--txn-id", transactionId];
```

Comment:
Oh interesting didn't know it works on historical transactions.

Maybe rename it to gasProfileExistingTransaction or something like that?

Or just leave room in this function to take a different input that also allows gas profiling new transactions.